frontend/src/resetPswd.js

```diff
@@ -0,0 +1,69 @@
+import './form.css';
+export function FormButton({size, display_name}) {
+    return (
+        <button
+            className = "form_btn"
+            height = {size}
+            width = {size}
+        >{display_name}</button>
+    );
+}
+
+export function Logo({size}) {
+    return (
+        <div>
+        <img
+            className = "logo"
+            src = './careerConnectLogo.png' 
+            alt = "logo"
+            width = {size}
+            height = {size}
+        />
+        </div>
+    );
+}
+
+export function Textbox({name, type, display_name}) {
+    return (
+        <div className = "textbox">
+            <input type = {type} name = {name} id = {name} placeholder = {display_name}/>    
```

Comment:
consider adding a label element here like this

```css
<label htmlFor={name}>{display_name}</label>
<input type = {type} name = {name} id = {name} placeholder = {display_name}/>  
```

Reply:
i did this inteionally because i think it looks better

Reply:
ok, that's fine!